src/Extend/View.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extend;
+
+use Flarum\Extension\Extension;
+use Illuminate\Contracts\Container\Container;
+use Illuminate\Contracts\View\Factory;
+
+class View implements ExtenderInterface
+{
+    private $addNamespaces = [];
+    private $replaceNamespaces = [];
+
+    /**
+     * Register a new namespace of laravel views.
```

Comment:
Let's use this opportunity to add more explanation about what a **L**aravel namespace is, and how the namespace can then be referenced in other views or controllers. :wink: